.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: publish-packages
+
+on:
+    push:
+        branches:
+            - master
+            - "**"
+
+jobs:
+    build-and-publish:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Initial checkout
+              uses: actions/checkout@v2
+              with:
+                  token: ${{ secrets.BOT_PUBLISH_TOKEN }}
+
+            - name: Configure CI Git User
+              run: |
+                  git config --global user.name '@fremtind-bot'
+                  git config --global user.email 'fremtind-bot@users.noreply.github.com'
+                  git remote set-url origin https://fremtind-bot:$BOT_PUBLISH_TOKEN@github.com/fremtind/jokul
+
+            #            - name: Checkout and pull branch
+            #              run: git checkout "${GITHUB_REF:11}" && git pull
+
+            - name: Install Packages
+              run: yarn install --frozen-lockfile && git config --list --global
+
+            - name: Authenticate with Registry
+              run: |
+                  yarn logout
+                  echo "registry=http://registry.npmjs.org/" >> .npmrc
+                  echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> .npmrc
+                  npm whoami
+              env:
+                  NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+#            - name: Publish packages
```

Comment:
Hvorfor er disse kommentert? For testing purposes?

Reply:
@joms jepp! Måtte teste at det kjørte med de nye endringene @lfbergee foreslo, men ville sørge for at den ikke prøvde å publisere fra denne branchen. Er endret tilbake nå 😊